docs/topics/web-openshift-troubleshoot.adoc

```diff
@@ -0,0 +1,132 @@
+. Hover over `Applications` along the left side of the console.
+. Click `Pods` from the options that appear.
+. Click the name of the pod that you wish to examine. To examine the current {WebName} pod, select the `rhamt-web-console-__POD_NAME__` that is in a `Running` state.
+. Click `Logs` from the options along the top.
+. To download the log file navigate to the top of the page and click `Download`.
+
+=== Using the OpenShift Client
+
+. Determine the pod name by executing `oc get pods` and search for the `web-console` entry with a `Running` status.
++
+[source,options="nowrap"]
+----
+$ oc get pod
+sNAME                        READY     STATUS      RESTARTS   AGE
+eap-builder-1-build         0/1       Completed   0          1d
+rhamt-postgresql-1-hfbdn    1/1       Running     0          1d
+rhamt-sso-1-build           0/1       Completed   0          1d
+rhamt-web-console-1-build   0/1       Completed   0          1d
+rhamt-web-console-1-vt7s5   1/1       Running     1          1d
+sso-1-wjl2n                 1/1       Running     1          1d
+----
++
+In the above example this is `rhamt-web-console-1-vt7s5`.
+
+. Use `oc logs` to examine the current pod's log.
++
+[source,options="nowrap",subs="+quotes"]
+----
+oc logs __POD_NAME__
+----
+
+. The output may be redirected to obtain a copy of the current log.
++
+[source,options="nowrap",subs="+quotes"]
+----
+oc logs __POD_NAME__ > ./rhamt-openshift-__POD_NAME__.log
+----
+
+== Common Issues with {WebNameTitle} OpenShift
+
+=== Executor Pod Throws `NoRouteToHostException`
+
+When accessing the executor pod's logs the following error is seen.
+
+[source,options="nowrap"]
+----
+13:44:03,501 SEVERE [org.jboss.windup.web.messaging.executor.ExecutorBootstrap] (main) Could not start messaging listener due to: Failed to connect to any server. Servers tried: [http-remoting://192.0.2.4:8080 (java.net.NoRouteToHostException: No route to host)]: javax.naming.CommunicationException: Failed to connect to any server. Servers tried: [http-remoting://192.0.2.4:8080 (java.net.NoRouteToHostException: No route to host)]
+[...]
+----
+
+*What it means:*
+
+A `NoRouteToHostException` indicates that this pod isn't able to connect to the {WebName} pod.
+
+*How to resolve it:*
+
+In a new deployment this is expected, as the executor starts before the {WebName}. Otherwise, examine the {WebName} pod and resolve any errors seen here. Once the {WebName} is running successfully this error should be resolved.
+
+=== Pod Takes Longer than 600 seconds to become available 
+
+When examining any pod's logs the following error is seen.
+
+[source,options="nowrap",subs="+quotes"]
+----
+error: update acceptor rejected rhamt-web-console-executor-1: pods for rc 'rhamt/__POD_NAME__' took longer than 600 seconds to become available
```

Comment:
Isn't is likely that the cause of that is that the web console itself failed to start?

Reply:
I saw the same message on the web console pod as well, thus the attempt at making this one a bit more generic; however, if this is seen on the web console does it mean something different? Can this be enhanced further?

Reply:
I reproduced this by creating three projects, and deploying the `empty-dir` template without changing the defaults. The first project started up successfully, but in both of the other two projects only the postgresql pod successfully started. 

For both of the other two projects the remaining pods, the web console and the executor, have `-deploy` in their title with an error. This is the error that was reported here. In this case I believe the deploy pod is the one that was failing, and it never was able to create the others?

If I attempt a redeployment of these pods it then proceeds to the insufficient resources error.